docs/Build/Meson_build.md

```diff
@@ -1,29 +1,61 @@
 # Build and Installation Instructions with Meson
 
+Meson is a modern and fast build system with a lot of features. You can
+find its documentation at [mesonbuild.com](https://mesonbuild.com/).
+
 ## Install Dependencies
 
-Fedora:
-```bash
-dnf install meson qt5-qtbase-devel rtaudio-devel "pkgconfig(jack) help2man"
-```
+=== "Fedora"
 
-Debian/Ubuntu:
-```bash
-apt install meson build-essential qtbase5-dev librtaudio-dev libjack-jackd2-dev help2man
-```
+    ```bash
+    dnf install meson qt5-qtbase-devel rtaudio-devel "pkgconfig(jack)" help2man
+    ```
 
-MacOS with brew (not tested):
-```bash
-brew install meson qt rt-audio jack help2man
-```
+=== "Debian/Ubuntu"
+
+    ```bash
+    apt install meson build-essential qtbase5-dev librtaudio-dev libjack-jackd2-dev help2man
+    ```
+
+=== "MacOS"
+
+    ```bash
+    brew install meson qt rt-audio jack help2man
+    ```
+
+    Meson might not find qt when installed with brew. But brew tells you to set
```

Comment:
Just a suggestion - I think the core of the issue is that qt is by default not symlinked to `/usr/local` for some presumably valid reasons. I'd guess that that's the reason meson is not finding it? If so, maybe the following would be more accurate:

`Qt installed with brew is not symlinked to`/usr/local` by default. Brew tells you (...)`

Reply:
If qt doesn't want to live in `/usr/local` it should append its pkgconfig directory to PKG_CONFIG_PATH. So your first suggestion looks fine to me. Could you test it?

Reply:
Sorry if I wasn't clear - I'm not contradicting my previous suggestion (I think), but rather wanted to clarify the reason for not finding qt - it's not an issue with meson, or qt install, but rather brew's decision to not link it by default. OTOH it's fine to leave your comment as is, totally up to you.

Reply:
It's kind of strange. Because other libraries work out of the box.